.github/workflows/documentation.yml

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: 2023 Berkay Bozkurt <resitberkaybozkurt@gmail.com>
+
+name: documentation
+
+on: [push, pull_request, workflow_dispatch]
```

Comment:
If I am not mistaken, this workflow will directly update the documentation at https://amosproj.github.io/amos2023ws06-sales-lead-qualifier/. Therefore we should only let in run on pushes to main, so that e.g. a push to feature/abc will not change our documentation for the entire project.

Reply:
It is currently running for each branch but only updates the github pages, when changes are made on `main` branch. So I believe, we can check if anything is wrong before affects the documentation before merging to `main` and publishing. Please check `documentation.yml : 34`.

Reply:
Then it's all good